### DISCUSSION TITLE
Regarding Your Data Collection Policy

### DISCUSSION THREAD

**Discussion by Umigishi-Aoi** [speechbrain/speechbrain#2687] [Q&A] [answered]
Dear Speech Brain Team,

Thank you very much for creating such a wonderful product.

I'm looking for a Speech-to-Text API capable of speaker diarization and came across Speech Brain. I'm very interested in using this product, but I have a question.

I would like to confirm whether or not using this product will result in the collection of audio data as training data for Speech Brain.

As we are developing a product that prioritizes user privacy, I would appreciate it if you could clarify this point.

If this service can be run entirely on my own server without any data transmission, I believe it would be suitable for our purposes.

Thank you for your time and consideration.

**Comment by asumagic** [comment, marked as answer]
SpeechBrain doesn't have telemetry and won't upload any data you infer on.

This is not legal advice, but we have no data collection policy, as we provide software rather than a service. You can check the source to make sure.

Network access will basically only occur e.g. to fetch remote models on HuggingFace (or sometimes in training scripts to download datasets and such), but you can also load a model locally.  
If your deployment allows, you could look into restricting internet access at a container/VM level as an additional safeguard.

**Reply by Adel-Moumen** [reply]
To clarify, SpeechBrain is NOT a company. There is no legal structure behind the project. The repository is under Apache 2.0, and as long as you comply with the License, then you can do whatever you want with the project. Additionally, there's no such concept as a "product". We don't try to make any money on top of the project. Everything is free to use.
